<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>897</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Παραγγελίες</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="2" alignment="Qt::AlignLeft">
     <widget class="QLineEdit" name="searchInput">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1500000</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Αναζήτηση</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QComboBox" name="fromSelector">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="6" alignment="Qt::AlignRight">
     <widget class="QLabel" name="dateLabel">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ημερομηνία:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="6">
     <widget class="QPushButton" name="addItem">
      <property name="text">
       <string>Προσθήκη Προϊόντος</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="createOrder">
      <property name="text">
       <string>Δημιουργία Παραγγελίας</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QComboBox" name="toSelector">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="fromLabel">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Από τμήμα:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6" alignment="Qt::AlignRight">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="10" column="0" colspan="7">
     <widget class="QTableWidget" name="orderView">
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>1</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Μονάδα Μέτρησης</string>
       </property>
       <property name="background">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>250</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Όνομα Προϊόντος</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QLabel" name="toLabel">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Προς τμήμα:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QPushButton" name="deleteItem">
      <property name="text">
       <string>Διαγραφή Προϊόντος</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3" alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="searchButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>19</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Αναζήτηση</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Μενού</string>
    </property>
    <widget class="QMenu" name="administrator">
     <property name="title">
      <string>Διαχειριστής</string>
     </property>
     <addaction name="changeProducts"/>
     <addaction name="separator"/>
     <addaction name="adminPasswordChange"/>
    </widget>
    <addaction name="administrator"/>
    <addaction name="openOrder"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="changeProducts">
   <property name="text">
    <string>Αλλαγή Προϊόντων</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Αλλαγή Κωδικού</string>
   </property>
  </action>
  <action name="adminPasswordChange">
   <property name="text">
    <string>Αλλαγή Κωδικού</string>
   </property>
  </action>
  <action name="openOrder">
   <property name="text">
    <string>Άνοιγμα παραγγελίας</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Έξοδος</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>orderView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
